projects/core/tokens/dialog-options.ts

```diff
@@ -0,0 +1,30 @@
+import {InjectionToken, ValueProvider} from '@angular/core';
+
+import {TuiDialogOptions} from '../interfaces';
+
+export type TuiDialogDefaultOptions = Omit<TuiDialogOptions<unknown>, 'data'>;
+
+export const TUI_DIALOG_DEFAULT_OPTIONS: TuiDialogDefaultOptions = {
+    size: `m`,
+    required: false,
+    closeable: true,
+    dismissible: true,
+    label: ``,
+    header: ``,
+};
+
+export const TUI_DIALOG_OPTIONS = new InjectionToken<TuiDialogDefaultOptions>(
```

Comment:
Most of the tokens `TUI_SOMETHING_OPTIONS` we place nearby the entity which this token customises.

Could you move this file `projects/core/tokens/dialog-options.ts` => `projects/core/components/dialog/dialog-options.ts`, please.

Reply:
Actually, I put it there because a similar token `TUI_NOTIFICATION_OPTIONS` was in that directory too

Reply:
@nsbarsukov

Reply:
@festimprebreza I think it's better to move next to the files, as the `tokens ` subpackage will be removed in the future